pkg/nsm/apis/nsmconnect/nsmconnect.proto

```diff
@@ -0,0 +1,126 @@
+syntax = "proto3";
+
+package nsmconnect;
+
+import "github.com/ligato/networkservicemesh/netmesh/model/netmesh/netmesh.proto";
+
+message Label {
+    map<string ,string> selector = 1;
+}
+
+message Metadata {
+    string name = 1;
+    string namespace = 2;
+    Label labels = 3;
+}
+
+message KernelInterface {
+   Metadata metadata = 1;
+// Interface specific parameters go here.
+}
+
+message VhostUserInterface{
+    Metadata metadata = 1;
+// Interface specific parameters go here.
+}
+
+message MemInterface {
+    Metadata metadata = 1;
+// Interface specific parameters go here.
+}
+
+message SRIOV {
+    Metadata metadata = 1;
+// Interface specific parameters go here.
+}
+
+enum InterfaceType {
```

Comment:
These should be CAPITALIZED_WITH_UNDERSCORES

Reply:
Does this still need to be addressed?